src/modules/getCounts.js

```diff
@@ -0,0 +1,10 @@
+import { updateLikes } from './manageLikes.js';
+
+export const displayCounts = (myPokes) => {
+  const counts = document.querySelector('.myPokes');
+  counts.innerText = `${myPokes} Pokes`;
```

Comment:
I think it will be better to refer to the plural of pokemon as "pokemons".
The short work poke is normally used to refer as thinks to use in pokemon world. Like "pokeballs", "poke-fruit", erc...